wiki/mapping/basic-lighting.md

```diff
@@ -38,21 +39,92 @@ though they may be in slightly different positions or not present in some cases.
 * **Boost Event:** Changes lighting colors between two pairs of colors. Off uses the first pair of colors.
   On uses the second (boost) pair of colors. There is no fade or flash option for this event.
 
-## Lighting Events
-Each lighting type can make use of four different lighting events.
-
-* **On:** Turns a light on that will stay on until you place a different event type.
-* **Off:** Turning a light off can have as much impact as turning it on, or more.
-* **Flash:** Briefly flashes brighter than a standard "on" event then stays on until you place a different event type.
-* **Fade:** Briefly flashes brighter than a standard "on" event then fades off after a couple of seconds.
+In Weave and later environments, or v3 environments, the set of objects
```

Comment:
`In Weave and later environments, or v3 environments,` suggest `In environments that use the v3 Group Lighting System (Weave and later),`